### PR TITLE
Link to alpakka keyword in scaladex

### PR DESCRIPTION
**Pull request by 2m**

### CODE REVIEW

File: docs/src/main/paradox/external-connectors.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
 # External Connectors
 
-Connectors that are maintained by other projects.
+[![alpakka]][alpakka-scaladex] Feel free to tag your project with *alpakka* keyword in Scaladex for easier discoverability.
```
Comment (raboof) [inline docs/src/main/paradox/external-connectors.md]: perhaps we should include a link to the docs that explain how to tag projects in Scaladex?

Comment (2m) [inline docs/src/main/paradox/external-connectors.md]: Do you mean scaladex docs?

Comment (raboof) [inline docs/src/main/paradox/external-connectors.md]: Yes :)

Comment (2m) [inline docs/src/main/paradox/external-connectors.md]: Have you seen a doc page in scaladex that explains this?

Comment (raboof) [inline docs/src/main/paradox/external-connectors.md]: Ha no... perhaps we should propose an update to their README?

Comment (2m) [inline docs/src/main/paradox/external-connectors.md]: Hm. I am not sure. The management page for a project in scaladex is quite self explanatory.

**Pull request merged**